### PR TITLE
resolve metaform issues

### PR DESCRIPTION
**Pull request by codev0**
- fix field with null meta form issues

### CODE REVIEW

File: website/frontend/.eslintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["airbnb-typescript", "airbnb/hooks", "prettier"],
-  "plugins": ["react", "import"],
+  "plugins": ["import"],
```
Comment (nikitayutanov) [inline website/frontend/.eslintrc:3]: Why?

File: website/frontend/package.json [hunk 2]
```diff
@@ -48,12 +48,24 @@
...
+      {
+        "files": [
+          "**/*.stories.*"
+        ],
+        "rules": {
+          "import/no-anonymous-default-export": "off"
+        }
+      }
     ]
   },
```
Comment (nikitayutanov) [inline website/frontend/package.json:70]: Doesn't `eslintConfig` in the `package.json` gets overridden by `.eslintrc`?

File: website/frontend/.storybook/main.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    '@storybook/preset-create-react-app',
+  ],
+  framework: '@storybook/react',
+  core: {
+    builder: 'webpack5',
```
Comment (nikitayutanov) [inline website/frontend/.storybook/main.js:11]: Why `webpack 5` over default `@storybook/builder-webpack5`?

File: website/frontend/.eslintrc (hunk 1 above)
Comment (codev0) [reply website/frontend/.eslintrc]: restored

**Comment by nikitayutanov** [comment]
Main goal is to improve code readibility and quality.

Some quick observations:
- Decomposite large components and functions into smaller ones, organize them, etc
- Create functions/variables for constantly repeating ones
- Inner `__*varName*` variables should be incapsulated
- Tests: stick to naming conventions, orginize them

Something to think of:
- Metaform name and names of inner variables based on it. Is it really `MetaForm` and not `PayloadForm` or something?

File: .github/workflows/CI-build-website.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           node-version: 16.x
       - run: |
-          yarn install
+          yarn install --immutable
```
Comment (osipov-mit) [inline .github/workflows/CI-build-website.yml:19]: Why is it needed? yarn install runs with this flag on CI anyway

**Review by osipov-mit — CHANGES_REQUESTED** [review]

File: website/frontend/package.json [hunk 5]
```diff
@@ -104,6 +118,7 @@
     "stream-browserify": "3.0.0",
     "typescript": "4.6.3",
     "vercel": "24.1.0",
-    "web-vitals": "2.1.4"
+    "web-vitals": "2.1.4",
+    "webpack": "5.70.0"
```
Comment (nikitayutanov) [inline website/frontend/package.json:122]: No need for explicit webpack declaration

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**